flash

```diff
@@ -487,15 +487,12 @@ add_datetime() {
   # get current UTC date/time
   current_datetime=$(date -u)
 
-  ed "$1" <<EOF
-  /^bootcmd:.*/
-  a
-  - [ cloud-init-per, once, sdate_msg, echo, "Setting initial date/time to: $current_datetime" ]
-  - [ cloud-init-per, once, sdate, date, -s, "$current_datetime" ]
-.
-w
-q
-EOF
+  /usr/bin/sed -i .bak -e \
+  "/^bootcmd/a\\
```

Comment:
why you use two backslashes?

Reply:
Because I'm using double quotes (`"`), so you have to escape the backslash, otherwise it will be gone and you get a shell error.